sass/components/_global.scss

```diff
@@ -22,3 +22,8 @@ a {
     margin-bottom: 0;
   }
 }
+
+.ecl-page-header-standardised .ecl-content-language-switcher--standardised {
+  border-bottom: 1px solid $ecl-color-blue-25!important;
+  color: $ecl-color-grey-75;
+}
```

Comment:
Do we still need this? I was doing it in the SCSS below.

Reply:
Looks like leftover after rebasing. I will remove it.